sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/batch/BulkExecutorTest.java

```diff
@@ -43,7 +56,7 @@ public class BulkExecutorTest extends BatchTestBase {
 
     @Factory(dataProvider = "clientBuilders")
     public BulkExecutorTest(CosmosClientBuilder clientBuilder) {
-        super(clientBuilder);
+        super(clientBuilder.directMode());
```

Comment:
This test was using gateway client for testing, why are we changing to direct mode?
Also, this is not a good way to change it. A better way would be to actually change the dataProvider above. If you want to change to direct mode, then you can use this dataProvider - `clientBuildersWithDirectTcpSession`